src/test/java/io/jenkins/plugins/conventionalcommits/utils/ProjectTypeTest.java

```diff
@@ -105,4 +105,14 @@ public void isNotPythonProject() throws IOException {
     ProjectType projectType = new PythonProjectType();
     assertEquals(false, projectType.check(pyDir));
   }
+
+  @Test
+  public void isPythonProject_WithTOMLFile() throws IOException {
```

Comment:
Not a big deal but it seems weird to me to follow both snake case and camelcase syntax, so either  `isPythonProjectWithTOMLFile` or `is_python_project_with_TOML_file` the first one seems better as globally the current code base use camelcase

Reply:
I will update it at once

Reply:
Done @olblak